_sass/base.scss

```diff
@@ -92,6 +92,11 @@ figure {
   margin: 0;
 }
 
+figcaption {
```

Comment:
I think we need to decrease the font size for the caption and add some bottom margin, how about like this?

![image](https://user-images.githubusercontent.com/337710/54058367-8a9b5100-41ed-11e9-965a-8d6a52333fa0.png)

(previously looked like this:)
![image](https://user-images.githubusercontent.com/337710/54058389-99820380-41ed-11e9-8897-3de924f892ff.png)

Reply:
Updated the CSS -- how's [this](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/a3bfe3d67258c454bcac391015737f3e46963c6f/index.html) look?